examples/webpack/main.js

```diff
@@ -0,0 +1,31 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/licenses/publicdomain/
+
+// Hello world example for webpack.
+
+require('pdfjs-dist');
+
+var pdfPath = '../helloworld/helloworld.pdf';
+
+// It is also possible to disable workers via `PDFJS.disableWorker = true`
```

Comment:
Do we want to mention this, as it degrades performance? People will find out when they inspect the API.